pyanaconda/ui/gui/spokes/installation_progress.py

```diff
@@ -218,28 +208,34 @@ def _cycle_rnotes(self):
 
     def initialize(self):
         super().initialize()
+        # Disable the continue button.
+        self.window.set_may_continue(False)
 
-        if not conf.system.can_reboot and conf.target.is_hardware:
-            continueText = self.builder.get_object("rebootLabel")
-            continueText.set_text(_("%s is now successfully installed on your system and ready "
-                                    "for you to use!  When you are ready, reboot your system to start using it!"))
-            continueText.set_line_wrap(True)
-            self.window.get_continue_button().set_label(C_("GUI|Progress", "_Quit"))
+        # Set the label of the continue button.
+        if conf.target.is_hardware and conf.system.can_reboot:
+            continue_label = C_("GUI|Progress", "_Reboot System")
+        else:
+            continue_label = C_("GUI|Progress", "_Finish Installation")
```

Comment:
Do we need this is there are no spokes and nothing can interrupt the installation?

Reply:
`Finish Installation` will appear instead of `Reboot System` when you are not allowed to reboot the system. That includes live, image and dir installations.

Reply:
Ahh true. OK in that case I guess we need it :D

Reply:
in the live case, the label previously was 'Quit', not 'Finish Installation'...